docs/docs/pods.md

```diff
@@ -0,0 +1,583 @@
+
+    ```bash
+    http DELETE <ip>:<port>/v2/pods/simplepod
+    ```
+
+# Technical Overview
+
+Pods are are members of Groups.* A pod instance’s containers are launched together, atomically, via the Mesos LAUNCH_GROUP call. Containers in pods share networking namespace and ephemeral volumes.
+
+\* Pods cannot be modified by the `/v2/groups/` endpoint, however. Pods are modified via the `/v2/pods/` endpoint.
+
+## Networking
+Marathon pods only support the [Mesos containerizer](http://mesos.apache.org/documentation/latest/mesos-containerizer/), which simplifies networking among containers. The containers of each pod instance share a network namespace and can communicate over localhost. 
+
+Containers in pods are created with endpoints. Other applications communicate with pods by addressing those endpoints. If you specify a container network without a name in a pod definition, it will be assigned to this default network.
+
+In your pod definition you can declare a `host` or `container` network type. Pods created with `host` type share the network namespace of the host. Pods created with `container` type use virtual networking. If you specify the `container` network type, you must also declare a virtual network name in the `name` field. See the [Examples](link) section for the full JSON.
+
+## Ephemeral Storage
+Containers within a pod share ephemeral storage. You can mount volumes by different names on each container in the pod.
+
+## Pod Definitions
+Pods are configured via a JSON pod definition, which is similar to an [application definition](http://mesosphere.github.io/marathon/docs/application-basics.html). You must declare the resources required by each container in the pod, even if Mesos is isolating at a higher (pod) level.  See the [Examples](link) section for complete pod definitions.
+
+### Secrets
+
+Specify a secret in the `secrets` field of your pod definition. The argument should be the fully qualified path to the secret in the store.
+
+```
+{
+	"secrets": /fully/qualified/path/
+}
+```
+
+### Volumes
+
+Marathon pods support several types of volumes are supported: ephemeral, persistent local and persistent external. All volumes are defined at the pod level. Your pod definition must include a `volumes` field that specifies at least the name of the volume and a `volumeMounts` field that specifies at least the name and mount path of the volume.
+
+```json
+{
+	"volumes": [
+		{
+			"name": "etc"
+		}
+	]
+}
+```
+
+```json
+{
+	"volumeMounts": [
+		{
+			"name": "env",
+			"mountPath": "/mnt/etc"
+		}
+	]
+}
+```
+
+### Containerizers
+
+Marathon pods support the [Mesos containerizer](http://mesos.apache.org/documentation/latest/mesos-containerizer/). The Mesos containerizer [allows you to specify alternative container images, such as Docker](http://mesos.apache.org/documentation/latest/container-image/). [Learn more about running Docker containers on Marathon](http://mesosphere.github.io/marathon/docs/native-docker.html).
```

Comment:
IMO this will be confusing to many users.  It would be better to say that Marathon supports Mesos and Docker containers through the UCR.     It is a mistake to claim that it only supports the Mesos containerizer.